### DISCUSSION TITLE
Which argument should I use?

### DISCUSSION THREAD

**Discussion by BellZwei** [lllyasviel/stable-diffusion-webui-forge#513] [Q&A]
Hi,
I'm currently running SDUI forge without any arguments, however, when I launch it i get this message in the console:

Launching Web UI with arguments:
Total VRAM 12282 MB, total RAM 32686 MB
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4070 Ti : native
Hint: your device supports --pin-shared-memory for potential speed improvements.
Hint: your device supports --cuda-malloc for potential speed improvements.
Hint: your device supports --cuda-stream for potential speed improvements.

**Comment by HMRMike** [comment]
cuda-stream with pin-shared-memory don't seem to do anything useful in my use on RTX 3090, but I didn't see any negative issues yet as well. Go ahead and try these, maybe on 12GB there will be more significant impact for the better.
cuda-malloc seems to be too small of an improvement to care about.

**Comment by freecoderwaifu** [comment]
It's not like your PC will explode, try one by one and all at once. If your webUI is slower, doesn't start or features and extensions don't work, remove them. If it's faster, keep them. If it's the same, keep them or remove them. Especially try higher resolutions, higher batch sizes, and SD 1.5 vs SDXL. Benchmark times before and after, compare it/s and time taken per generation, before and after.

**Comment by rockiecxh** [comment]
I have the same card, and I've enabled these options, but didn't feel much difference.

**Comment by BellZwei** [comment]
After many tries with different models I didn't see any notable impact on the performance.
